src/main/java/uk/gov/hmcts/opal/controllers/AccountTransferController.java

```diff
@@ -22,7 +22,7 @@
 
 
 @RestController
-@RequestMapping("/api/accountTransfers")
+@RequestMapping("/api/accountTransfer")
```

Comment:
[Use lowercase separate words with hyphens for Path Segments [129]](https://hmcts.github.io/restful-api-standards/#129)